### PR TITLE
Converted README to markdown + added ToC 

### PR DESCRIPTION
**Pull request by TwiN**
I'm aware that the very first version of this document was in Markdown (only had two lines at the time, though), and was then replaced by what it currently is. However, I took it upon myself to re-convert it and improve it significantly for much easier readability. 

Furthermore, I fixed a couple of typos and restructured some sections.

Finally, I added a table of contents for ease of access.

## TL;DR

- Converted README to Markdown
- Renamed from `README.org` to `README.md`
- Fixed typos
- Added table of contents

### CODE REVIEW

**Comment by thecsw** [comment]
Hey! I have a couple of questions.
Why Markdown when we have Org-Mode?
And can you update the Readme with the pull request #37?
Thanks!

**Comment by TwiN** [comment]
Will do when I get home; and because I feel Markdown is more well-known and more pleasant to edit, which would make people more likely to contribute, though that is entirely my personal opinion.

**Comment by thecsw** [comment]
Yeah, as this project is gaining more attention and not a lot of people use emacs, we can switch to markdown. Please solve the merge conflicts and I will start merging!

**Comment by TwiN** [comment]
@thecsw Done.
